action.yml

```
@@ -14,6 +14,12 @@ inputs:
       ideas.
     default: stable-2.13
     required: true
+  controller-python-version:
+    description: >-
+      Controller Python version. Only used for integration tests and
+      ansible-core 2.12 or later when `target-python-version` is also
+      specified.
+    default:
```

Why isn't it `auto`?
```suggestion
    default: auto
```

---

Because `auto` is not a valid value, and the default should be "not specified".

---

I thought, maybe it'd become "unspecified" where it's invoked. My thinking was that maybe this name would be better API-wise.

---

Ok, changing the default value to `auto` sounds good to me.

---

Implemented in ada1bbe59ac5b7f2b0fd17de169443c540a56367.